### ISSUE TITLE
Loss curve

### ISSUE THREAD

**Issue by Lixia1221** [yhenon/pytorch-retinanet#240]
How to view the drop curve of the loss function during training
